frontend/src/component/Mention/Mention.jsx

```diff
@@ -20,7 +20,7 @@ const MentionWrapped = props => {
 const calculateUserMention = (props) => {
   const mention = {
     isToMe: false, // true: mention to me, false: mention to someone else
-    text: undefined // Text to display in the mention
+    text: 'UndefinedUser' // Text to display in the mention
```

Comment:
will this be translated later?

Reply:
I mean, is "UndefinedUser" supposed to be displayed?

Reply:
We can translate it

``UndefinedUser`` should not be displayed, it's a _security_ so we are sure that something is always displayed (even if we should always know the username cause we use an API call in the last resort)

Reply:
understood, you can resolve this thread